Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -1,374 +1,207 @@
+  border: var(--remotion-themed-border);
+  border-radius: var(--remotion-themed-border-radius);
   white-space: nowrap;
-  cursor: hand;
   cursor: pointer;
-  line-height: 1.25em;
-  height: 1.4em;
 }
 
-td.buttonCell span.DropDownMenuSelect
+div.DropDownMenuOptions
 {
-  /* only when display just arrow */
-  display: inline-block;
+  /* int max for highest z-index possible */
+  z-index: 2147483647;
+  position: absolute;
 }
 
-span.DropDownMenuSelect.disabled
+ul.DropDownMenuOptions
 {
-  cursor: default;
-  border: 1px solid #BCBCBC;
+  box-shadow: var(--remotion-themed-box-shadow);
+  padding: var(--remotion-themed-padding);
+  list-style: none;
+  background-color: var(--color-disabled-background);
 }
 
-span.DropDownMenuSelect span
+span.DropDownMenuItemIcon
 {
-  left: 0;
-  vertical-align: middle;
+  width: 1.25em;
 }
 
-span.DropDownMenuSelect a.DropDownMenuLabel
+/* style separator */
+span.DropDownMenuSeparator
 {
-  position: relative;
-  left: 0.1em;
-  top: 0.1em;
-  margin-right: 25px;
-  margin-right: calc(22px + 0.1em);
-  margin-left: 4px;
-  border-style: none;
-  background-color: Transparent;
-  color: Black;
-  font-weight: bold;
-  height: 1.2em !important;
+  display: flex;
+  line-height: var(--remotion-themed-separator-height);
+  margin: var(--remotion-themed-margin) 0;
+  background-color: var(--color-contrast-separator);
 }
 
-span.DropDownMenuSelect a.DropDownMenuLabel:focus
+div.DropDownMenuStatus
 {
-  outline: none;
-  text-decoration: underline;
+  overflow: hidden;
+  /* int max for highest z-index possible */
+  z-index: 2147483647;
+  /* position and dimension are set via javascript */
+  max-height: 4em;
+  max-width: 30em;
 }
 
-span.DropDownMenuContainer span.disabled a.DropDownMenuLabel
+/* TODO: rework indicator when new design available */
+div.DropDownMenuStatus > div
 {
-  color: Gray;
+  width: 100%;
+  padding: 0.25em;
+  border: 1px solid Black;
+  background-color: Lightyellow;
+  box-sizing: border-box;
 }
 
-span.DropDownMenuSelect a.DropDownMenuButton
+div.DropDownMenuStatus.loading > div
 {
-  display: inline-block;
-  position: absolute;
-  left: auto;
-  top: 0px;
-  bottom: 0;
-  right: 0px;
-  border-left: solid 1px #AAAAAA;
-  margin-right: 0;
+  background-image: url('../Image/sprite.svg#DropDownMenuRequestInProgressIndicator');
+  background-size: 16px 16px;
+  background-repeat: no-repeat;
+  background-position: 0.25em;
+  padding-left: calc(16px + 0.25em);
+  min-height: 20px;
 }
 
-span.DropDownMenuSelect.focus a.DropDownMenuButton,
-span.DropDownMenuSelect:hover a.DropDownMenuButton
+div.DropDownMenuStatus.error > div
 {
-  border-left: solid 1px #7CB2E6;
+  background-image: url('../Image/sprite.svg#DropDownMenuErrorIndicator');
+  background-size: 16px 16px;
+  background-repeat: no-repeat;
+  background-position: 0.25em;
+  padding-left: calc(16px + 0.5em);
+  min-height: 20px;
 }
 
-span.DropDownMenuSelect.nestedHover:hover a.DropDownMenuButton
+ul.DropDownMenuOptions li a
```

Comment:
the focused item gets an underline for the text.

Reply:
I added the underlining of the focused item, however, in the "Workstep Popup" design template there is no underline for the focused item, so I'm 100% sure whether it's correct or not.

Reply:
the styleguide did not conder the concept of "focus" so that's up to us now.